fitbenchmarking/fitbenchmarking
# Fix Installation not working on Release branch

#### Description of Work
This PR fixes the installation problem experienced in the issue. The error does not exist in master, but only on the release branch. Whether its worth creating a patch for this is TBD.

Fixes #753

#### Testing Instructions
Make sure you have python 3.9  installed first of all

1. Create a virtual environment `py -m venv env`
2. Activate the environment `.\env\Scripts\activate`
3. Clone the release branch `git clone -b release-0.1.x https://github.com/fitbenchmarking/fitbenchmarking.git`
4. Go into the fitbenchmarking directory and do `python -m pip install .[bumps,DFO,minuit,SAS]`. There should be an error.
5. Now repeat but for this branch. There should be no error.

Function: Does the change do what it's supposed to?

Tests: Does it pass? Is there adequate coverage for new code?

Style: Is the coding style consistent? Is anything overly confusing?

Documentation: Is there a suitable change to documentation for this change?
